lib/routes/emails.js

```diff
@@ -491,12 +494,15 @@ module.exports = (log, db, mailer, config, customs, push) => {
         const sessionToken = request.auth.credentials
         const uid = sessionToken.uid
 
-        if (! features.isSecondaryEmailEnabled(sessionToken.email)) {
-          return reply(error.featureNotEnabled())
-        }
+        return db.account(uid)
+          .then((account) => {
 
-        db.accountEmails(uid)
```

Comment:
Sorry, just to clarify why I asked that, it seems like we'd benefit from having a single canonical place to get any email address associated with the account, rather than hitting a different endpoint depending on which email we're after (and needing to know which endpoint to hit)...?

Feel free to ignore if it sounds like nonsense, I'm still learning about this feature.